docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
```

Comment:
Ok next time it might be easier to just morph this PR



> On Feb 28, 2015, at 12:04 AM, Zhinan Lin(Zen Lin) <notifications@github.com> wrote:
> 
> In docker/docker.go:
> 
> > @@ -111,7 +111,7 @@ func main() {
> >  			*flTls = true
> >  			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
> >  			if err != nil {
> > -				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
> > +				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
> @duglin @thaJeztah I have accepted suggestions from both of you, new a PR #11073 to commit the change.
> Thx very much.
> 
> —
> Reply to this email directly or view it on GitHub.
>